app/uk/gov/hmrc/selfservicetimetopay/controllers/ArrangementController.scala

```diff
@@ -51,11 +51,11 @@ object ArrangementController extends FrontendController {
 
```

Comment:
This method has not been checked since it is part of the 'your installment plan summary' story. It will be properly refactored then